contracts/RiskFund/RiskFund.sol

```diff
@@ -151,9 +136,14 @@ contract RiskFund is Ownable2StepUpgradeable, ExponentialNoError, ReserveHelpers
      * @dev Swap single asset to Base asset.
      * @param vToken VToken
      * @param comptroller comptorller address
+     * @param amountOutMin Minimum amount to receive for swap
      * @return Number of swapped tokens.
      */
-    function swapAsset(VToken vToken, address comptroller) internal returns (uint256) {
+    function swapAsset(VToken vToken, address comptroller, uint256 amountOutMin) internal returns (uint256) {
```

Comment:
instead of `vToken` we can take address of `underlying` directly.

Reply:
I use the `VToken` to get the comptroller which is used to index pool reserves which is why I am using the `VToken` here